kv/testHelpers.go

```diff
@@ -0,0 +1,24 @@
+package kv
+
+import "github.com/tobiasfamos/KVStore/stub"
+
+type TestHelper interface {
+	GetWorkingDirectory(store KeyValueStore) string
+	GetEmptyInstance() KeyValueStore
+}
+
+type StubTestHelper struct {
+}
+
+func (StubTestHelper) GetWorkingDirectory(store KeyValueStore) string {
+	return ""
+}
+
+func (StubTestHelper) GetEmptyInstance() KeyValueStore {
+	return new(stub.KvStoreStub)
```

Comment:
Moving stub into `kv` package as per above.
```suggestion
	return new(KvStoreStub)
```